src/app/savings/saving-account-actions/savings-account-transactions/savings-account-transactions.component.scss

```
@@ -27,3 +27,126 @@
   font-weight: 500;
   margin-bottom: 0.5rem;
 }
+
+/* Stepper styles */
+::ng-deep .mat-stepper-horizontal {
```

Stylelint (17.4.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)